### PR TITLE
Replace .aicoe.yaml with .prow.yaml

### PR DESCRIPTION
**Pull request by anishasthana**
This commit also removes the kustomize build check script as
the operate-first toolbox image now includes it.

Signed-off-by: Anish Asthana <anishasthana1@gmail.com>

### CODE REVIEW

**Comment by anishasthana** [comment]
cc @harshad16

File: .prow.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    decorate: true
+    max_concurrency: 1
+    run_if_changed: ".*yaml"
+    skip_report: false
+    context: aicoe-ci/prow/kustomize-build
+    spec:
+      containers:
+        - image: quay.io/operate-first/opf-toolbox:v0.3.1
+          command:
+            - "test-kustomize-build"
```
Comment (harshad16) [inline .prow.yaml:12]: is this  to run from the image ? if so, isn't the file kept at /usr/local/bin/ ?

**Review by tumido — COMMENTED** [review]
/lgtm

File: .prow.yaml (hunk 1 above)
Comment (anishasthana) [reply .prow.yaml:12]: /usr/local/bin is already in the $PATH right? so we can run like this from anywhere?

Comment (tumido) [reply .prow.yaml:12]: yes, it's run from the image, the script is in `/usr/local/bin` which is in `PATH`, though I've discovered some permissions issue, I'll fix that asap so it's callable directly

Comment (harshad16) [reply .prow.yaml:12]: i saw the permission issue as well in my local, that why i asked :)

Comment (anishasthana) [reply .prow.yaml:12]: Thanks @tumido

**Review by maulikjs — APPROVED** [review]

**Pull request merged**